print-isakmp.c

```diff
@@ -2386,7 +2442,7 @@ ikev2_n_print(netdissect_options *ndo, u_char tpay _U_,
 	ND_TCHECK_SIZE(p);
 	ikev2_pay_print(ndo, NPSTR(ISAKMP_NPTYPE_N), GET_U_1(p->h.critical));
 
-	showspi = 1;
+	showspi=0;
```

Comment:
I changed the initial value of `showspi` in order to make the cases of the switch-statement less noisy. For the existing cases, the value of `showspi` should be unchanged, unless I made a mistake. For the new cases, It's not clear to me according to which criterion the SPI should be printed or not.

Wouldn't it make more sense to always print the SPI if it is present? I.e., to change the line
```
if (showspi && spi_size) {
```
below to

```
if (spi_size) {
```

Maybe you can enlighten me @mcr?   You introduced it in 650e708afb8c6b20d49dfec794a238eac9c70736.

Reply:
I removed the spaces around the assignment because the two assignments below don't have them either. Alternatively, I could add surrounding spaces to all three assignments.